circom-types/src/plonk/verification_key.rs

```diff
@@ -20,7 +20,7 @@ where
     /// The number of public inputs
     #[serde(rename = "nPublic")]
     pub n_public: usize,
-    /// Proof element Z    
+    /// Proof element Z
```

Comment:
These comments are all the same and wrong?

Reply:
fixed